### PR TITLE
Update input.md

### PR DESCRIPTION
**Pull request by utkarshbajaj**
Updated input.md file, as given in the issue no #859 in https://github.com/appsmithorg/appsmith.
Please merge/add Hacktoberfest-approved/approve It to Make it count towards Hacktoberfest.

### CODE REVIEW

**Comment by utkarshbajaj** [comment]
@Nikhil-Nandagopal @mohanarpit

File: widget-reference/input.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ description: >-
 
 # Input
 
+![compressed](https://user-images.githubusercontent.com/44445191/94912021-a7c8fc00-04c4-11eb-84c9-32a6f2390296.gif)
```
Comment (hetunandu) [inline widget-reference/input.md:10]: Upload this gif in the assets folder

**Review by hetunandu — CHANGES_REQUESTED** [review]
Please go through the docs contribution guide and follow how assets need to be uploaded and referenced. You can find the guides [here](https://github.com/appsmithorg/appsmith/blob/release/contributions/docs/CONTRIBUTING.md)

**Comment by utkarshbajaj** [comment]
I have done the changes according to the documentation. 
@hetunandu @mohanarpit @Nikhil-Nandagopal

**Review by Nikhil-Nandagopal — APPROVED** [review]

File: widget-reference/input.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ description: >-
 
 # Input
 
+![](/.gitbook/assets/input-widget.gif)
```
Comment (hetunandu) [inline widget-reference/input.md:10]: Few changes needed in the gif
- The property pane is not visible in the gif. Please make sure all properties are shown in the gif.
- The size is too big. Please try to get the size of the file down to 2 mb so it loads faster for a user when viewing documentation

**Review by hetunandu — CHANGES_REQUESTED** [review]
@utkarshbajaj This looks great! Just some minor changes in the gif and you should be GTG

File: widget-reference/input.md (hunk 2 above)
Comment (utkarshbajaj) [reply widget-reference/input.md:10]: Sure, I'll do them.

Comment (utkarshbajaj) [reply widget-reference/input.md:10]: Screenshot 2020-10-05 at 3 20 15 PM
When I go inside widgets, these properties are no longer available

Comment (utkarshbajaj) [reply widget-reference/input.md:10]: Ill make a gif where I click from here and go inside ?

Comment (hetunandu) [reply widget-reference/input.md:10]: Yea this is a nice starting point

**Review by hetunandu — APPROVED** [review]

**Comment by hetunandu** [comment]
Thanks for your patience to make this perfect @utkarshbajaj 😄 Really appreciate this contribution and looking forward to any other contributions as well. Make sure you go through our list of issues again

**Comment by utkarshbajaj** [comment]
> Thanks for your patience to make this perfect @utkarshbajaj 😄 Really appreciate this contribution and looking forward to any other contributions as well. Make sure you go through our list of issues again

Sure!

**Pull request merged**